pom.xml

```diff
@@ -184,6 +184,8 @@
 
         <!-- Forbidden API related properties -->
         <forbiddenapis-junit.path>forbidden-junit.txt</forbiddenapis-junit.path>
+
+        <hibernate-validator-parent.basedir>.</hibernate-validator-parent.basedir>
```

Comment:
this is removed in later commits with adding of `maven-remote-resources-plugin`
at first I was thinking about adding it to every pom, but I got lazy :) and done it a bit differently in the end. This `maven-remote-resources-plugin` do something similar to `maven-dependency` unpack, and places those configs in target so each module should have all configs available in target folder ...